package.json

```diff
@@ -15,7 +15,7 @@
     },
     "dependencies": {
         "@types/sarif": ">=2.1.1 <=2.1.2",
-        "axe-core": "^3.2.2",
+        "axe-core": "^4.0.1",
```

Comment:
We actually don't generally update this (that's why it's still back at 3.2.2), since we don't want to force users to update axe-core in order to update our library (it would have been better for us to use a `peerDependency` for this, but we decided we didn't want to make a breaking change to fix it). You will want to update this to claim *support* for using 4.x as well as 3.x though; I think I'd suggest `"^3.2.2 || ^4.0.0"`